build.rb

```diff
@@ -110,100 +110,113 @@ def name_object(var_prefix, language_map)
+      identifiers: [
+        {
+          scheme: 'wikidata',
+          identifier: membership[:party].value,
+        },
+      ],
+    }
+  end.uniq
+
+  areas = membership_rows.map do |membership|
+    {
+      name: membership.name_object('district_name', LANGUAGE_MAP),
+      id: membership[:district].value,
+      identifiers: [
+        {
+          scheme: 'wikidata',
+          identifier: membership[:district].value,
+        },
+      ],
+      type: {
+        'lang:en_CA': 'federal electoral district of Canada',
+             'lang:fr_CA': 'circonscription électorale fédérale canadienne',
       },
-    ],
-  }
-end.uniq
+      parent_id: 'Q16',
+    }
+  end.uniq
 
-areas = membership_rows.map do |membership|
-  {
-    name: membership.name_object('district_name', LANGUAGE_MAP),
-    id: membership[:district].value,
+  area_country = {
+    name: {
+      'lang:en_CA': 'Canada',
+           'lang:fr_CA': 'Canada',
+    },
+    id: 'Q16',
     identifiers: [
       {
         scheme: 'wikidata',
-        identifier: membership[:district].value,
+        identifier: 'Q16',
       },
     ],
     type: {
-      'lang:en_CA': 'federal electoral district of Canada',
-      'lang:fr_CA': 'circonscription électorale fédérale canadienne',
+      'lang:en_CA': 'Country',
+           'lang:fr_CA': 'Pays',
     },
-    parent_id: 'Q16',
   }
-end.uniq
-
-area_country = {
-  name: {
-    'lang:en_CA': 'Canada',
-    'lang:fr_CA': 'Canada',
-  },
-  id: 'Q16',
-  identifiers: [
+
+  memberships = membership_rows.map do |membership|
     {
-      scheme: 'wikidata',
-      identifier: 'Q16',
-    },
-  ],
-  type: {
-    'lang:en_CA': 'Country',
-    'lang:fr_CA': 'Pays',
-  },
-}
-
-memberships = membership_rows.map do |membership|
-  {
-    id: membership[:statement].value,
-    person_id: membership[:item].value,
-    on_behalf_of_id: membership[:party].value,
-    area_id: membership[:district].value,
-    start_date: membership[:start]&.value,
-    end_date: membership[:end]&.value,
-    role_code: membership[:role].value,
-    role: membership.name_object('role', LANGUAGE_MAP),
-  }.reject { |_, v| v.to_s.empty? }
-end
+      id: membership[:statement].value,
+      person_id: membership[:item].value,
+      on_behalf_of_id: membership[:party].value,
+      area_id: membership[:district].value,
+      start_date: membership[:start]&.value,
+      end_date: membership[:end]&.value,
+      role_code: membership[:role].value,
+      role: membership.name_object('role', LANGUAGE_MAP),
+    }.reject { |_, v| v.to_s.empty? }
+  end
 
-all_data = {
-  persons: persons,
-  organizations: organizations,
-  areas: [area_country] + areas,
-  memberships: memberships,
-}
+  all_data = {
+    persons: persons,
+    organizations: organizations,
+    areas: [area_country] + areas,
+    memberships: memberships,
+  }
 
-puts JSON.pretty_generate(all_data)
+  output_filename.open('w') do |f|
+    f.write(JSON.pretty_generate(all_data) + "\n")
```

Comment:
As `output_filename` is actually a `Pathname` here, and not just a string (see earlier comment), this could just be: `output_filename.write(JSON.pretty_generate…)`